### ISSUE TITLE
Ttags and std/top

### ISSUE THREAD

**Issue by ragerdl** [acl2/acl2#349]
I find myself including std/top a lot, because that's the library I like to use (looking up specific libraries  takes time).  Something I dislike about this practice, though, is that now I have to add a `cert.acl2` file to the directory just so that ttags will be allowed (or, at least, that's my impression).  Might we consider two versions of std/top, one that requires ttags and one that doesn't?  Maybe `std/top-fast` and `std/top` (or `std/top-slow` and `std-top`)?

Or is there another work-around?

**Comment by MattKaufmann** [comment]
Hi, David --

I just did the following in a build that I think is up-to-date (it was
yesterday), and I got no ttags report.  So I wonder what's going on.

ACL2 !>(include-book "std/top" :dir :system)

Summary
Form:  ( INCLUDE-BOOK "std/top" ...)
Rules: NIL
Time:  4.29 seconds (prove: 0.00, print: 0.00, other: 4.28)
 "/v/filer4b/v11q002/acl2space/acl2/acl2/books/std/top.lisp"
ACL2 !>

-- Matt
   Date: Tue, 24 Feb 2015 14:54:48 -0800
   From: "David L. Rager" <notifications@github.com>
   Reply-To: acl2/acl2 <reply@reply.github.com>


   I find myself including std/top a lot, because that's the library I like to use (looking up specific libraries  takes time).  Something I dislike about this practice, though, is that now I have to add a `cert.acl2` file to the directory just so that ttags will be allowed (or, at least, that's my impression).  Might we consider two versions of std/top, one that requires ttags and one that doesn't?  Maybe `std/top-fast` and `std/top` (or `std/top-slow` and `std-top`)?

   Or is there another work-around?

   ---
   Reply to this email directly or view it on GitHub:
   https://github.com/acl2/acl2/issues/349
